TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -9,14 +9,27 @@
 using TypeCobol.Compiler.Nodes;
 using TypeCobol.Compiler.Parser;
 using System.Text.RegularExpressions;
+using TypeCobol.Compiler.Directives;
 using TypeCobol.Compiler.Scanner;
 
 namespace TypeCobol.Compiler.Diagnostics
 {
     public class CrossCompleteChecker : AbstractAstVisitor
     {
+        public CrossCompleteChecker(TypeCobolOptions compilerOptions = null)
```

Comment:
I think it is better to not allorw null object for this parameter. If CrossCompleteChecker requires TypeCobolOptions, we should not allow default options instead. Use `public CrossCompleteChecker([NotNull] TypeCobolOptions compilerOptions)`.

Reply:
fixed
https://github.com/TypeCobolTeam/TypeCobol/tree/828d955ed380f37e3a26def80e2d534f11fb26d7